tests/testExamples/Test_with_variables_favorite.js

```diff
@@ -0,0 +1,26 @@
+module.exports = function (dslName) {
+    return {
+        name: 'test',
+        description: 'test',
+        type: 'dsl',
+        is_favorite: true,
```

Comment:
not sure we need this example of a file
we can use every other file and play with is_favorite param